app/util/analyticsUtils.js

```diff
@@ -25,9 +25,20 @@ export function addAnalyticsEvent(event) {
  *
  * @param {number|string} GTMid Google Tag Manager id
  *
+ * @param MTM_URL
  * @return string
  */
-export function getAnalyticsInitCode(GTMid) {
+export function getAnalyticsInitCode(GTMid, MTM_URL) {
+  if (MTM_URL) {
+    return `<!-- Matomo Tag Manager -->
+      <script type="text/javascript">
+      var _mtm = _mtm || [];
+      _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+      var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+      g.type='text/javascript'; g.async=true; g.defer=true; g.src='${MTM_URL}'; s.parentNode.insertBefore(g,s);
+      </script>
+      <!-- End Matomo Tag Manager -->`;
+  }
```

Comment:
Should we really support both at the same time? I'd suggest to use else if instead of if

Reply:
I believed if there is a return in the 'if' section, the other 'ifs' do not have a chance. (ESLint also gives an error with this modification: "unnecessary else after return".) So only the first true statement will have an effect.
Would you like me to do it anyway?